src/deployment/ios.md

```diff
@@ -161,21 +140,36 @@ app's icons:
 1. Verify the icon has been replaced by running your app using
    `flutter run`.
 
-## Create a build archive with Xcode
-
-This step covers creating a build archive and uploading
-your build to App Store Connect.
+## Create a build archive and upload to App Store Connect
 
 During development, you've been building, debugging, and testing
 with _debug_ builds. When you're ready to ship your app to users
 on the App Store or TestFlight, you need to prepare a _release_ build.
-At this point, you might consider [obfuscating your Dart code][]
-to make it more difficult to reverse engineer. Obfuscating
-your code involves adding a couple flags to your build command.
 
-In Xcode, configure the app version and build:
+### Update the app's build and version numbers
 
-1. In Xcode, open `Runner.xcworkspace` in your app's `ios` folder.
+The default version number of the app is `1.0.0`.
+To update it, navigate to the `pubspec.yaml` file
+and update the following line:
+```yaml
+version: 1.0.0+1
+```
+The version number is three numbers separated by dots,
+such as `1.0.0` in the example above, followed by an optional
+build number such as `1` in the example above, separated by a `+`.
+
+Both the version and the build number may be overridden in
+`flutter build ipa` by specifying `--build-name` and `--build-number`,
+respectively.
+
+In iOS, `build-name` uses `CFBundleShortVersionString`
+while `build-number` uses `CFBundleVersion`.
+Read more about iOS versioning at [Core Foundation Keys][]
+on the Apple Developer's site.
+
+You may also override the `pubspec.yaml` build name and number in Xcode:
+
+1. Open `Runner.xcworkspace` in your app's `ios` folder.
 1. Select **Runner** in the Xcode project navigator, then select the
```

Comment:
This part should have been documented with the `Update the app's build and version numbers`, which I moved to this section.